src/components/FacetOptionFormatter/FacetOptionFormatter.js

```diff
@@ -0,0 +1,27 @@
+import PropTypes from 'prop-types';
+
+import { OptionSegment } from '@folio/stripes/components';
+
+import styles from './FacetOptionFormatter.css';
+
+const propTypes = {
+  option: PropTypes.object,
```

Comment:
I think that it should be required. You use this component only as `formatter` for MCL, `formatter` works for each element of `dataOptions`. In your MCL you have no any empty element in `dataOptions`. In this case, you also don't need condition below

Reply:
We can have empty elements in MCL when using pagination.
For example, if you're viewing page 2, your options array will have length `pageSize * 2`, but the first `pageSize` number of elements in array will be empty, and the rest will be actual objects